app/client/cypress/integration/home/links.spec.js

```diff
@@ -17,4 +17,28 @@ describe('Homepage', () => {
     cy.findByText('National').click();
     cy.url().should('include', '/national');
   });
+  it(`"Data" button links to the data page`, () => {
```

Comment:
Add blank line between tests.
```suggestion
  });
  
  it(`"Data" button links to the data page`, () => {
```